### PR TITLE
Fb1138

### PR DESCRIPTION
**Pull request by spireken**
I put the build in the time_and_sales_properties_window Dropbox folder.

### CODE REVIEW

File: Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp [hunk 1]
```diff
@@ -35,7 +36,7 @@ time_and_sales_properties_dialog::time_and_sales_properties_dialog(
   m_body->setStyleSheet("background-color: #F5F5F5;");
   auto layout = new QVBoxLayout(m_body);
   layout->setContentsMargins(scale_width(8), scale_height(10), scale_width(8),
-    scale_height(8));
+    0);
```
Comment (kranar) [inline Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp]: Fit this on the line above.

File: Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp [hunk 2]
```diff
@@ -206,95 +207,18 @@ time_and_sales_properties_dialog::time_and_sales_properties_dialog(
...
-  apply_button->set_style(generic_button_default_style);
-  apply_button->set_hover_style(generic_button_hover_style);
-  apply_button->set_focus_style(generic_button_focused_style);
-  buttons_layout_4->addWidget(apply_button);
-  buttons_layout_4->setStretchFactor(apply_button, 26);
-  buttons_layout_4->addStretch(8);
-  auto ok_button = new flat_button(tr("OK"), this);
-  ok_button->connect_clicked_signal(
+  buttons_widget->connect_ok_signal(
     [=] { accept(); });
```
Comment (kranar) [inline Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp]: Fit this on the line above as well. Only continue a line if there's no reasonable way to fit it in 80 columns.

File: Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp [hunk 3]
```diff
@@ -206,95 +207,18 @@ time_and_sales_properties_dialog::time_and_sales_properties_dialog(
...
-  ok_button->setFont(generic_button_font);
-  ok_button->set_style(generic_button_default_style);
-  ok_button->set_hover_style(generic_button_hover_style);
-  ok_button->set_focus_style(generic_button_focused_style);
-  buttons_layout_4->addWidget(ok_button);
-  buttons_layout_4->setStretchFactor(ok_button, 26);
-  buttons_layout->addLayout(buttons_layout_4);
-  buttons_layout->setStretchFactor(buttons_layout_4, 100);
+  buttons_widget->connect_cancel_signal(
+    [=] { reject(); });
```
Comment (kranar) [inline Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp]: Same here.

**Comment by spireken** [comment]
The 'layout->...' line won't fit.

**Pull request merged**